prime-router/metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform.yml

```diff
@@ -90,11 +90,24 @@ elements:
 
   # SFT-1-2
```

Comment:
Like RADx MARS, these changes intend to preserve the behavior for CA, despite formerly overridden elements being adjusted or deleted.